MDANSE/Tests/UnitTests/Analysis/test_scattering.py

```diff
@@ -105,10 +108,15 @@ def test_dcsf(traj_info, qvector_grid):
     assert path.isfile(temp_name + ".mda")
 
     result_file = os.path.join(result_dir, f"dcsf_{traj_info[0]}.mda")
-    with h5py.File(temp_name + ".mda") as actual,  h5py.File(result_file) as desired:
-        keys = [i for i in desired.keys() if any([j in i for j in ["f(q,t)", "s(q,f)"]])]
+    with h5py.File(temp_name + ".mda") as actual, h5py.File(result_file) as desired:
+        keys = [
+            i for i in desired.keys() if any([j in i for j in ["f(q,t)", "s(q,f)"]])
+        ]
```

Comment:
Slightly more explicit expectations? Also similarly elsewhere?
```suggestion
        keys = [
            key for key in desired.keys() 
            if any(key.startswith(j) for j in ["f(q,t)", "s(q,f)"])
        ]
```